### DISCUSSION TITLE
Is there a way to clear/close all active toasts?

### DISCUSSION THREAD

**Discussion by underwood** [usernotnull/tall-toasts#381] [General]
Is there a way to close any active toasts?

**Comment by underwood** [comment]
Something like: toast()->clear();

**Comment by usernotnull** [comment]
Not at the moment. You'd need to index toasts so that u'd get that index to remove it in the next requests.
Usually the use case is extremely rare because toasts are usually meant to be fleeting, so feature was not developed.
